editor/project_export.cpp

```diff
@@ -1202,23 +1245,58 @@ ProjectExportDialog::ProjectExportDialog() {
 	// Script export parameters.
 
 	updating_script_key = false;
+	updating_enc_filters = false;
 
 	VBoxContainer *script_vb = memnew(VBoxContainer);
 	script_vb->set_name(TTR("Script"));
 	script_mode = memnew(OptionButton);
 	script_vb->add_margin_child(TTR("Script Export Mode:"), script_mode);
 	script_mode->add_item(TTR("Text"), (int)EditorExportPreset::MODE_SCRIPT_TEXT);
 	script_mode->add_item(TTR("Compiled"), (int)EditorExportPreset::MODE_SCRIPT_COMPILED);
-	script_mode->add_item(TTR("Encrypted (Provide Key Below)"), (int)EditorExportPreset::MODE_SCRIPT_ENCRYPTED);
+	script_mode->add_item(TTR("Encrypted (Provide key on Security tab)"), (int)EditorExportPreset::MODE_SCRIPT_ENCRYPTED);
 	script_mode->connect("item_selected", callable_mp(this, &ProjectExportDialog::_script_export_mode_changed));
+	sections->add_child(script_vb);
+
+	VBoxContainer *sec_vb = memnew(VBoxContainer);
+	sec_vb->set_name(TTR("Security"));
```

Comment:
Should we name it "Encryption" instead of "Security"? (Open question, I don't have a strong opinion, but we might be expected to guarantee more security than what whitebox encryption offers if we name it this way.)